macros/utils/cross_db_utils/type_bigint.sql

```diff
@@ -0,0 +1,7 @@
+{% macro type_bigint() %}
```

Comment:
We have a data_types folder on our utils, maybe it's a better fit (all the type macros)

Reply:
I just wanted to make it clear that this is created to support the deprecation of dbt-utils.
But sure I can move it over there..